docs/testing/regression-checklist/mapping.json

```
@@ -13,20 +13,20 @@
 		"testFile": [ "extensions/all.md" ]
 	},
 	{
-		"regex": "extensions/blocks/contact-form*",
+		"regex": "extensions/blocks/contact-form.*",
```

Since it's not anchored and not capturing, there's no point in ending it with `.*`.
```suggestion
		"regex": "extensions/blocks/contact-form",
```